### PR TITLE
Port view_conflicts.js, view_errors.js and view_include_docs.js into elixir

### PR DESCRIPTION
**Pull request by jjrodrig**
## Overview
This PR ports following tests into Elixir test suite:
- view_conflicts.js
- view_errors.js
- view_include_docs.js

## Testing recommendations

```
 make elixir tests=test/elixir/test/view_conflicts_test.exs
 make elixir tests=test/elixir/test/view_errors_test.exs
 make elixir tests=test/elixir/test/view_include_docs_test.exs
```

## Related Issues or Pull Requests
N/A

## Checklist

- [X] Code is written and works correctly
- [X] Changes are covered by tests
- [ ] Any new configurable parameters are documented in `rel/overlay/etc/default.ini`
- [ ] A PR for documentation changes has been made in https://github.com/apache/couchdb-documentation

### CODE REVIEW

File: test/elixir/test/view_errors_test.exs [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+      views: %{
+        reduce_too_big: %{
+          map: "function (doc) {emit(doc.integer, doc.integer)};",
+          reduce:
+            "function (keys, values) { var chars = []; for (var i=0; i < 1000; i++) {chars.push('wazzap');};return chars; };"
+        }
+      }
+    }
+
+    {:ok, _} = create_doc(db_name, design_doc2)
```
Comment (dottorblaster) [inline test/elixir/test/view_errors_test.exs:248]: Is this because the test would fail on getting an `:error` tagged tuple?

**Review by dottorblaster — CHANGES_REQUESTED** [review]
Impressive! Sorry for the delay about getting on this review 😄 

Could you tag the tests with `@moduletag kind: :single_node` please? We are tagging test modules by setup in order to run them clustered by setup type from the fastest to the slowest.

File: test/elixir/test/view_errors_test.exs (hunk 1 above)
Comment (jjrodrig) [reply test/elixir/test/view_errors_test.exs:250]: Yes, no :error is expected here. It will fail if we get something different.

**Comment by jjrodrig** [comment]
Thanks @dottorblaster for the review. 
I've added the @moduletag kind to these tests

**Review by dottorblaster — APPROVED** [review]
Brilliant!

**Pull request merged**